### DISCUSSION TITLE
Does OcarSlicer support high speed 3d printer？

### DISCUSSION THREAD

**Discussion by huminlian** [OrcaSlicer/OrcaSlicer#5563] [General]
I‘m a Bambulab P1P user and I usually use the Bambu Studio for model slicing. Does OcarSlicer can used for high speed [core xy printer](https://kingroon.com/products/kingroon-klp1-corexy-3d-printer-with-klipper-firmware), like Creality K1C and SVO8?

**Comment by igiannakas** [comment]
Yes it does...

**Comment by foreachthing** [comment]
@huminlian, if you can setup your printer in the slicer, it will support it. _Maybe_ there is no pre-made profile for your printer. There was non for our Ultimaker 2 and I made and submitted one (for everyone to use).
